### PR TITLE
1letter/bs5 dashboard

### PR DESCRIPTION
**Pull request by 1letter**
- add bootstrap5 nav tabs to dashboard.pt
- compatible with Plone6 Barceloneta Theme

### CODE REVIEW

**Comment by agitator** [comment]
Tabs look good so far :-)
What do you think about the columns? lg-4 md-2 ... ?
And with https://getbootstrap.com/docs/5.0/layout/grid/#row-columns or specific col-sizes?

**Comment by 1letter** [comment]
@agitator Ok, i have add a two column layout,also in plone.app.portlets for the management view. i think 4 columns are to small.

File: plone/app/layout/dashboard/dashboard.pt [hunk 1]
```diff
@@ -1,50 +1,70 @@
...
+                  href="${context/@@plone_portal_state/navigation_root_url}/@@manage-dashboard?_authenticator=${view/auth_token}"
+                  class="nav-link"
+                  i18n:translate="label_edit">Edit</a>
+            </span>
+          </nav>
+
+
+          <div
+              id="dashboard"
+              class="row row-cols-1 row-cols-md-2">
```
Comment (agitator) [inline plone/app/layout/dashboard/dashboard.pt:49]: maybe ads a  gy-2 to balance x/y gutter?

**Comment by agitator** [comment]
thx, I also went for a max of 2-cols in a client project.

**Pull request merged**